index.bs

```diff
@@ -663,6 +669,8 @@ agent MUST run the following steps:
    congestion control algorithms that optimize for |congestionControl|, as allowed by
    [[!RFC9002]] [section 7](https://datatracker.ietf.org/doc/html/rfc9002#section-7),
    then set |congestionControl| to `"default"`.
+1. Let |rateControlFeedbackMinInterval| be 1000 milliseconds.
+1. Let |rateControlFeedback| be a [=new=] {{WebTransportRateControlFeedback}}.
```

Comment:
These stack variables are unused in this algorithm. They probably need to be [[InternalSlots]] on the wt instead.

Reply:
I added some internal slots to fix this.